README.md

```diff
@@ -23,57 +22,35 @@ from those installed system wide.
 Install dependencies
 ```sh
 $ pip install -r requirements.txt
+# Also install wine and curl.
 ```
 
 Build fonts
 ```
-$ make
-```
-
-Add legacy kern table
-```
-$ make kerntable
+$ make -j<number of logical cores in your CPU>
```

Comment:
Excellent simplification!
We can probably drop the `-j…`, as this is an optimisation but confuses what is otherwise an extremely simple command "just type `make`"